packages/block-library/src/image/block.json

```diff
@@ -85,7 +85,6 @@
 	"supports": {
 		"anchor": true,
 		"color": {
-			"__experimentalDuotone": true,
```

Comment:
This is something I found by reviewing the duotone code (not related to this PR): https://github.com/WordPress/gutenberg/blob/70198b9811c1bdc99579e272a72af3dae0ec9138/lib/class-wp-duotone-gutenberg.php#L281

Reply:
I think this is needed to enable the filter controls in the block toolbar.